website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -48,12 +48,16 @@ Attribute                        | Description
 ---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 `hosted-json-state-download-url` | A URL from which you can download the state data in a [stable format](/terraform/internals/json-format) appropriate for external integrations to consume. Only available if the state was created by Terraform 1.3+.
 `hosted-state-download-url`      | A URL from which you can download the raw state data, in the format used internally by Terraform.
+`hosted-json-state-upload-url`   | A URL to which you can upload the state data, in a [stable format](/terraform/internals/json-format) appropriate for external integrations to consume. Only available if the JSON state content has not been uploaded.
+`hosted-state-upload-url`        | A URL to which you can upload the state data, in the format used internally by Terraform. Only available if the state content has not been uploaded.
 `modules`                        | Extracted information about the Terraform modules in this state data. Populated asynchronously.
 `providers`                      | Extracted information about the Terraform providers used for resources in this state data. Populated asynchronously.
+`provisional`                    | A boolean flag that denotes a state version that was created but has not yet marked as the current version for a workspace.
```

Comment:
```suggestion
`provisional`                    | A boolean flag indicating that Terraform Cloud created a state version but did not yet finalize it and mark it as the current version for the workspace.
```

Reply:
The only thing I'll omit here is the "not yet finalize it". Provisional state versions can be finalized.